README.md

```diff
@@ -8,30 +8,48 @@ This repository holds the ReStructuredText (RST) source, and other files, for us
+### Running automated tests
 
-Run `nox --list` from the repository root to view available sessions.
+This project includes a `nox` configuration to automate tests, checks, and other functions.
+You can use these automated tests to help you verify changes before you submit a PR.
 
-Run `nox` with no arguments to execute the default sessions.
+1. Install `nox` using `python3 -m pip install nox` or your distribution's package manager.
+2. Run `nox --list` from the repository root to view available sessions.
 
-## Running the spelling check
+Each `nox` session creates a temporary environment that installs all requirements and runs the test or check.
+This means you only need to run one command to perform the test accurately and consistently.
+Here are some `nox` sessions you can run:
 
-This repository uses [`codespell`](https://github.com/codespell-project/codespell) to check for common spelling mistakes in the documentation source.
+* Run `nox` with no arguments to run all available sessions.
+* Run `nox -s clone-core` to clone required parts of the `ansible/ansible` repository as mentioned in the [local build documentation](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#setting-up-your-environment-to-build-documentation-locally).
+* Run `nox -s "checkers(rstcheck)"` to ensure there are no syntax errors in the reStructuredText source files.
+* Run `nox -s "checkers(docs-build)"` or `nox -s "checkers(docs-build)" -- --relaxed` to verify the docs build.
 
-Run `nox -s spelling` to check spelling.
+> The `nox -s "checkers(docs-build)"` session cleans the generated docs after it runs.
+> If you want to view the generated HTML in your browser, you should build the documentation locally.
 
-Run `nox -s spelling -- -w` to correct spelling errors.
+## Checking spelling
 
-When `codespell` suggests more than one word as a correction, run `nox -s spelling -- -w -i 3` to select an option.
+Use [`codespell`](https://github.com/codespell-project/codespell) to check for common spelling mistakes in the documentation source.
 
-## Updating the dependencies
+* Run `nox -s spelling` to check spelling.
+* Run `nox -s spelling -- -w` to correct spelling errors.
+
+> When `codespell` suggests more than one word as a correction, run `nox -s spelling -- -w -i 3` to select an option.
+
+## Updating dependencies
+
+Run `nox -e pip-compile` to update project dependency versions.
+
+> This session requires Python 3.10.
+
+If you do not have Python 3.10 installed, you can use root-less podman with a Python 3.10 image as follows:
 
-To update dependencies, you can use `nox -e pip-compile`. Since this requires Python 3.10, this might not work in your environment if you do not have Python 3.10 installed. In that case, you can use root-less podman with a Python 3.10 image:
 ```bash
 podman run --rm --tty --volume "$(pwd):/mnt:z" --workdir /mnt python:3.10 bash -c 'pip install nox ; nox -e pip-compile'
```

Comment:
While we're at it, I'd like to change this to use the fully qualified container name. If you prefer to handle this in a separate PR, that's fine with me as well.

```suggestion
podman run --rm --tty --volume "$(pwd):/mnt:z" --workdir /mnt docker.io/library/python:3.10 bash -c 'pip install nox ; nox -e pip-compile'
```

Reply:
I think it's fine to do that here.